uai2017/python/precreate-spc-profiles.py

```diff
@@ -0,0 +1,124 @@
+import openreview
+import requests
+import csv
+import argparse
+import re
+
+## Handle the arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('file',help="the csv file containing the SPC names and email addresses")
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+if args.username!=None and args.password!=None:
+    client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+else:
+    client = openreview.Client(baseurl=args.baseurl)
+
+spc_member_info = []
+with open(args.file) as f:
+    reader = csv.reader(f)
+
+    for row in reader:
+        spc_member_info.append((row[0],row[1],row[2]))
+
+tildematch = re.compile('~.+')
+for u in spc_member_info:
+    email = u[0].strip()
+    first = u[1].strip()
+    last = u[2].strip()
+
+    try:
+        member_groups = client.get_groups(member=email)
```

Comment:
You should start checking if the email exists or not. 

If the email exists, check the first member, that should be the tilde group. 

Why do you get groups by member?